base_comment_template/migrations/15.0.1.1.0/pre-migration.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr, "ALTER TABLE base_comment_template ADD COLUMN IF NOT EXISTS models text"
+    )
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE base_comment_template template
```

Comment:
Have you tried if this query is working?

Reply:
Yes, I tested it locally.